tests/test_H0.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import pytest
+import os
+from datetime import datetime, timedelta
+
+from RadClass.RadClass import RadClass
+from RadClass.H0 import H0
+import tests.test_data as test_data
+
+# initialize sample data
+start_date = datetime(2019, 2, 2)
+delta = timedelta(seconds=1)
+timestamps = np.arange(start_date,
+                       start_date + (test_data.timesteps * delta),
+                       delta).astype('datetime64[s]').astype('float64')
+
+live = np.full((len(timestamps),), test_data.livetime)
+sample_val = 1.0
+spectra = np.full((len(timestamps), test_data.energy_bins),
+                  np.full((1, test_data.energy_bins), sample_val))
+# setting up for rejected null hypothesis
+spectra[int(test_data.timesteps/2):] = 100.0
```

Comment:
Python 3 integer division
```suggestion
spectra[test_data.timesteps//2:] = 100.0
```

It also make may sense to define this as a variable of when you expect the trigger to occur